client/components/select-dropdown/index.jsx

```diff
@@ -214,6 +225,7 @@ class SelectDropdown extends Component {
 		const selectedText = this.getSelectedText();
 		const selectedIcon = this.getSelectedIcon();
 		const selectedSecondaryIcon = this.getSelectedSecondaryIcon();
+		const positionSelectedSecondaryIconOnRight = this.getPositionSelectedSecondaryIcon();
```

Comment:
This seems to be doing

```jsx
const { positionSelectedSecondaryIconOnRight } = this.props
```

Reply:
Oh yeah, you're right 👌

Reply:
I added this change [in this commit](https://github.com/Automattic/wp-calypso/pull/74846/commits/d33221c07a0742857ba8c970b9302afc54988407).